documentation/README.md

```diff
@@ -149,7 +149,7 @@ localhost:8000
 If you don't have 'postgres' installed on your machine you can install with the following packages:
 ```bash
 /> sudo apt-get install postgresql-contrib
-/> sudo apt-get install postgresql-server-dev-9.3
+/> sudo apt-get install postgresql-server-dev-10
```

Comment:
Why did you upgrade the DB. The issue was not to upgrade the DB, but to upgrade to django 2.2. with python 3.8 ?